flink-end-to-end-tests/test-scripts/common_s3.sh

```diff
@@ -68,8 +68,8 @@ function s3_setup {
   trap s3_cleanup EXIT
 
   cp $FLINK_DIR/opt/flink-s3-fs-hadoop-*.jar $FLINK_DIR/lib/
-  echo "s3.access-key: $ARTIFACTS_AWS_ACCESS_KEY" >> "$FLINK_DIR/conf/flink-conf.yaml"
```

Comment:
let's either change these to use the variables defined at the top (`AWS_REGION`, `AWS_ACCESS_KEY`, `AWS_SECRET_KEY`), or remove said variables since they are unused.

Reply:
I don't know all the implication of changing/removing these variables in bash scripts ;-)
It is also orthogonal to the change here, I would like to leave this for a dedicated cleanup setp